src/customize-pop/accessibility-seeing.md

```diff
@@ -0,0 +1,187 @@
+
+![Screen Part](/images/accessibility-settings/screen-part.png)
+
+The zoomed area of the screen renders to a static portion that corresponds with the selected drop-down option.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/screen-part.webm" />
+</video>
+
+**Magnifier Extends Outside of the Screen**
+
+When `Magnifier extends outside of the screen` is enabled, magnified content continues to scroll when the cursor moves toward the edges of the screen.
+
+**Disabled**
+
+![Screen Part](/images/accessibility-settings/extend-disabled.png)
+
+**Enabled**
+
+![Screen Part](/images/accessibility-settings/extend-enabled.png)
+
+**Keep Magnifier Cursor Centered**
+
+When `Keep magnifier cursor centered` is enabled, the cursor stays in a fixed position in the center unless moving along the screen's edges. Magnified content always scrolls beneath the cursor.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/cursor-centered.webm" />
+</video>
+
+**Magnifier Cursor Pushes Contents Around**
+
+When `Magnifier cursor pushes contents around` is enabled, magnified contents do not move until the cursor pushes against the edges of the magnified area.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/pushes-content.webm" />
+</video>
+
+**Magnifier Cursor Moves with Contents**
+
+Magnified content moves with the mouse, but the mouse does not remain in a fixed position on the screen. This provides indication of the general area of the mouse's position on the un-magnified screen.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/moves-with-content.webm" />
+</video>
+
+#### Crosshairs
+
+Crosshairs help useres locate their mouse while navigating zoomed content.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/crosshairs.webm" />
+</video>
+
+| Option | Function |
+|---------|----------|
+| Overlaps mouse cursor | Check this setting to layer the crosshairs over the cursor. Uncheck this setting end the crosshair lines before they reach the cursor. |
+| Thickness | Adjust the thickness of the crosshair's lines between 1 and 100 pixels. |
+| Length | Set the length of the visible crosshair. |
+| Color | Launch a color picker to customize the crosshair's color and transparency. |
+
+#### Color Effects
+
+Color effects modify the color of zoomed content. These settings assist users with photophobia or when using the computer under poor lighting conditions.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/color.webm" />
+</video>
+
+| Option | Function |
+|--------|----------|
+| White on black | Toggle on to invert white and black colors: White becomes black, black becomes white, dark grays become lighter, lighter colors and grays become darker. Color hues do not change. |
+| Brightness | Adjust the overall brightness of the zoomed content. `Low` and `High` are reversed when `White on black` is toggled to the on position. |
```

Comment:
> toggled to the on position

Minor nitpick/suggestion, but "enabled" seems like a simpler way to say this.

Reply:
Updated with your suggested wording.